### PR TITLE
Adjust entrypoint setup

### PR DESCRIPTION
**Pull request by nadirhamid**
This PR adds to the existing SSL checks in entrypoint.sh. I have added environment variable SSL_ENABLED which will let users switch SSL setting on and off. This patch is useful to any user who may not want to deploy in SSL mode.

### CODE REVIEW

File: templates/cloudformation-template.yaml [hunk 1]
```diff
@@ -89,11 +89,11 @@ Parameters:
...
   Domain:
     Type: String
-    Default: ''
+    Default: 'example.org'
 
   Email:
     Type: String
-    Default: ''
+    Default: 'test@example.org'
   AccessKey:
```
Comment (roddylindsay) [inline templates/cloudformation-template.yaml:97]: can these be blank by default and dynamically required based on another flag HTTPSIsEnabled in the template (disabled by default)?

File: templates/cloudformation-template.yaml [hunk 2]
```diff
@@ -564,7 +564,8 @@ Resources:
           Statement:
           - Effect: Allow
             Action: ['elasticloadbalancing:DeregisterInstancesFromLoadBalancer', 'elasticloadbalancing:DeregisterTargets',
-              'elasticloadbalancing:Describe*', 'elasticloadbalancing:RegisterInstancesWithLoadBalancer',
+                 Image: envelopsound/earshot:v0.2
```
Comment (roddylindsay) [inline templates/cloudformation-template.yaml:567]: why is this in here?

**Review by roddylindsay — CHANGES_REQUESTED** [review]

File: templates/cloudformation-template.yaml (hunk 1 above)
Comment (roddylindsay) [reply templates/cloudformation-template.yaml:97]: also, these need to be applied to the other templates as well

**Comment by nadirhamid** [comment]
@roddylindsay  Updates made as suggested. I have updated the CF stack and the documentation. Please have a look.

Thanks

**Review by roddylindsay — APPROVED** [review]

**Pull request merged**